core/controllers/suggestion_test.py

```diff
@@ -475,3 +702,218 @@ def test_accept_question_suggestion(self):
             suggestion_to_accept['suggestion_id'])
         last_message = thread_messages[len(thread_messages) - 1]
         self.assertEqual(last_message.text, 'This looks good!')
+
+
+class TopicSuggestionTests(test_utils.GenericTestBase):
+
+    AUTHOR_EMAIL = 'author@example.com'
+
+    def setUp(self):
+        super(TopicSuggestionTests, self).setUp()
+        self.signup(self.OWNER_EMAIL, self.OWNER_USERNAME)
+        self.signup(self.AUTHOR_EMAIL, 'author')
+
+        self.owner_id = self.get_user_id_from_email(self.OWNER_EMAIL)
```

Comment:
Can we call this user admin? "Owner" confuses it with exploration owner which is what it is generally used as.

Reply:
Done